cms/static/js/factories/manage_users.js

```diff
@@ -9,7 +9,7 @@ function(_, gettext, ManageUsersAndRoles) {
             var local_messages = _.extend({}, messages);
             local_messages.alreadyMember.title = gettext('Already a course team member');
             local_messages.deleteUser.messageTpl = gettext(
-                'Are you sure you want to delete {email} from the course team for “{container}”?'
+                'Are you sure you want to delete <%= email %> from the course team for “<%= container %>”?'
```

Comment:
@nedbat You can read the earlier comments to first get some context.  The ``safe_interpolate`` method that I was hoping to introduce for JavaScript would look like the examples below.

However, for this case where the the template engine was being used for string interpolation, I don't have a good recommendation.  The below syntax is clearly too verbose as an alternative, and I don't even know if ``gettext()`` could be properly scraped if we did want it. 
 
Example Template string:
```
<%= 
    safe_interpolate(
        gettext( 'Are you sure you want to delete {email} from the course team for “{container}”?', {
            email: email,
            container: container
        })
%>
```
Example Template string with HTML tags:
```
<%= 
    safe_interpolate(
        gettext( 'Are you sure you want to delete {strongStart}{email}{strongEnd} from the course team for “{container}”?', {
            email: email,
            container: container,
            strongStart: HTML('<strong>'),
            strongEnd: HTML('</strong>')
        })
%>
```

If I understand @cahrens correctly, the way this was working was by redefining the interpolate tag from ``<!%`` and ``%>`` to ``{`` and ``}`` in the template.   The original code that did this worked for interpolation, but I'm not sure if the final escaping ever happens.  Also, I don't see what would be done in the case of my example above, if the strong tag needed to be inserted into the text.

Maybe this is simple and my brain is over complicating it?  Help if you have any good suggestions.  Thank you.